### DISCUSSION TITLE
Clarifiction on use cases

### DISCUSSION THREAD

**Discussion by gcolonna12** [ezzcodeezzlife/dalle2-in-python#30] [Q&A] [answered]
Great project!

Just wondering to what extent you have tried to send request via this library. According to the Terms of Service, it's not allowed to programmatically generate images for the time being. I'm wondering to what extent you have tried and tested this tool and how sure we can be that our accounts won't get banned.

**Comment by ezzcodeezzlife** [comment, marked as answer]
I have tested it a lot and didnt get banned or anything. This package has all the limitations of the web user interface. 

One good use-case is generating more than just 4 images with a couple lines of code. This was time consuming in the web UI.
